corehq/apps/reports/static/reports/js/aggregate_user_status.js

```diff
@@ -21,6 +29,16 @@ hqDefine("reports/js/aggregate_user_status", function () {
             if (customTooltip) {
                 chart.tooltipContent(customTooltip);
             }
+
+            // Add scrollbar for large datasets.
+            // Multiplication factor for chart width is chosen so that chart is readable
+            dataLength = data['values'].length
+            if (dataLength > 120) {
```

Comment:
How was the number 120 chosen (was it through visually testing the report)? Also, how does this constant hold up when a user has different browser zoom levels?

Reply:
Correct. The number was chosen such that graph is readable visually for days greater than 120. This is applicable for chart width multiplication factor (15) too. 

> Also, how does this constant hold up when a user has different browser zoom levels?

Good question here. So for period with 120 or less, the graph (bar size) automatically adjusts based on zoom.
For other case, the graph just gets zoomed out or zoom in based on browser zoom levels.